app/views/reports/invoice_reports/new.html.erb

```diff
@@ -13,7 +13,7 @@
   <div class="row">
     <div class="columns">
       <%= f.label :project_id %>
-      <%= f.collection_select :project_id, Project.active, :id, :name, include_blank: "Select a project" %>
+      <%= f.collection_select :project_id, Project.active, :id, :name, {}, {multiple: true, class: "multiselect"} %>
```

Comment:
Does select2 add a blank option? Just want to make sure we're not getting rid of that.

Reply:
Select2 have default text that says "Make a Selection(s)" set in the js, which will show when there's no selection. The reason I got rid of this is because since there can be multiple, and it was sending the blank selection through the params which wasn't needed, and sometimes it would show as a selection.